src/site/content/en/handbook/inclusion-and-accessibility/index.md

```diff
@@ -72,7 +71,7 @@ In the rare case that you need to omit an accessibility detail to better convey
 ## Writing about people
 There are several excellent online guides for using inclusive language when writing about groups of people. The [Content Guide](https://content-guide.18f.gov/inclusive-language/) from 18F (a U.S. government office that helps other agencies improve their user experience) is a great place to start.
 
-To be more gender inclusive, use _they/them_ for singular personal pronouns instead of _he/him_ and _she/her_. However, if you can use a plural pronoun instead, that's ideal.
+To be more gender inclusive, use _they/them_ for singular personal pronouns instead of _he/him_ and _she/her_. However, it's ideal to make it plural.
```

Comment:
This introduces a pronoun-antecedent error. (There's no singular noun to which "it" can refer.)